public/js/controllers/extras_controller.js

```diff
@@ -0,0 +1,24 @@
+import { Controller } from 'stimulus'
+
+export default class extends Controller {
+  copyTextToClipboard (textElement) {
+    var copySupported = document.queryCommandSupported('copy')
+    var textContent = textElement.srcElement.parentNode.textContent.trim()
+    if (copySupported === true) {
+      var copyTextArea = document.createElement('textarea')
+      copyTextArea.value = textContent
+      document.body.appendChild(copyTextArea)
+      copyTextArea.select()
+      try {
+        document.execCommand('copy')
+        window.alert('Copied text ' + textContent + ' to clipboard')
```

Comment:
I am open to suggestions for anything that would be better than this alert?

Reply:
I tend to like sites that gently flash "Copied text to clipboard" or similar, either on top of the copied text with a somewhat opaque background, or in a little blob of text above the button.  but just pulsing/flashing the button to acknowledge the click would be good too.